README.md

```diff
@@ -307,6 +307,57 @@ $ sudo systemctl start station
 $ sudo systemctl status station
 ```
 
+## API
+
+### `Core({ cacheRoot?: String, stateRoot?: String })`
+
+Returns `Core`.
```

Comment:
Let's make it more clear that `Core` is a class and the constructor must be called with the `new` keyword.

For example:

```suggestion
### `new Core({ cacheRoot?: String, stateRoot?: String })`

Creates a new instance of `Core`.
```